src/pages/email_flow_events/edit-email-flow-event-page.js

```diff
@@ -15,9 +15,10 @@ import React from 'react'
 import { connect } from 'react-redux';
 import { Breadcrumb } from 'react-breadcrumbs';
 import T from "i18n-react/dist/i18n-react";
-import EmailFlowEventForm from '../../components/forms/email-flow-event-form';
+import EmailFlowEventForm from '../../components/forms/email-flow-event-form/email-flow-event-form';
```

Comment:
if u rename the email-flow-event-form.js to index.js
then u could omit this path and leave it as originally was

Reply:
done